src/lib.rs

```diff
@@ -45,6 +45,11 @@ use std::path::PathBuf;
 use std::sync::Mutex;
 use term::{self, color, Attr, StderrTerminal};
 
+#[cfg(feature = "failure-bt")]
+mod backdoortrace;
+#[cfg(feature = "failure-bt")]
+pub use backdoortrace::print_failure_backtrace;
```

Comment:
Did you intentionally leave out `backdoortrace` here? If you didn't, you can just do `pub use backdoortrace::*;`. It will only expose `pub` items.

Reply:
yea i figured people didn't really need the joke module name in their path when they want to use this feature.

Reply:
Oh no, what I was referring to is the function `backdoortrace`, not the module.

Reply:
Now that I notice that they are called the same, I realize that it might be rather confusing to have both names in the root of the crate, doing different things depending on context. So we should either leave it as-is or rename the func if we want to re-export it in the root of the module as well.